### PR TITLE
create/update all sql file

### PR DESCRIPTION
**Pull request by superpaul**
Like in 2.5.0 tar packages I've created a single sql schema file

### CODE REVIEW

File: azkaban-sql/src/sql/create-all-sql-3.0.0.sql [hunk 1]
```diff
@@ -0,0 +1,217 @@
+CREATE TABLE active_executing_flows (
+	exec_id INT,
+	host VARCHAR(255),
+	port INT,
+	update_time BIGINT,
```
Comment (logiclord) [inline azkaban-sql/src/sql/create-all-sql-3.0.0.sql]: You need to merge update sql script as well. https://github.com/azkaban/azkaban/tree/master/azkaban-sql/src/sql/update* ?

Comment (superpaul) [inline azkaban-sql/src/sql/create-all-sql-3.0.0.sql]: merged updates on a separate files instead.

File: azkaban-sql/src/sql/create-all-sql-3.0.0.sql [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+	options LONGBLOB NOT NULL,
+	primary key(exec_id, job_name)
+);
+
+CREATE TABLE execution_flows (
+	exec_id INT NOT NULL AUTO_INCREMENT,
+	project_id INT NOT NULL,
+	version INT NOT NULL,
+	flow_id VARCHAR(128) NOT NULL,
+	status TINYINT,
```
Comment (logiclord) [inline azkaban-sql/src/sql/create-all-sql-3.0.0.sql]: I meant it will be great to add executor_id column in DDL itself.

**Comment by suvodeep-pyne** [comment]
Closed as out of date. Please reopen if you think that this PR is still relevant.

**Pull request closed without merging**